backend/stacktraces/enhancer.go

```diff
@@ -87,6 +89,14 @@ func (n NetworkFetcher) fetchFile(ctx context.Context, href string) ([]byte, err
 		return nil, e.New("status code not OK")
 	}
 
+	if res.Header.Get("Content-Encoding") == "br" {
```

Comment:
Hmm I'll have to double check on my side as well, but I think this might actually be a bug in the Reflame CDN if we respond with a `br` encoded response when your request doesn't have a `Accept-Encoding` header with `br` in it.

Will add this to the backlog but I might not be able to get to it for a while since most browsers do support brotli so include it in accept-encoding. Appreciate y'all putting in this workaround in the meantime!